js/src/modals/AddAddress/addAddress.js

```diff
@@ -89,7 +89,7 @@ export default class AddAddress extends Component {
         onClick={ this.onAdd }
         ref='addButton'
       />
-    ]);
+    ];
```

Comment:
Ummm... I actually fixed this in my modals update branch (along with a ton of others, we didn't seem to be as vigilant as we should have), however re-doing it here.